File: modulea/src/main/java/com/bxfox/modulea/MainActivity.java
package com.bxfox.modulea;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.alibaba.android.arouter.facade.annotation.Route;

@Route(path = "/a/module_a_activity")
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_module_aactivity);
    }
}

File: modulea/src/main/java/com/bxfox/modulea/MainActivity.kt
package com.bxfox.modulea

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.alibaba.android.arouter.facade.annotation.Route
import com.alibaba.android.arouter.launcher.ARouter

@Route(path = "/moduleA/main")
class MainActivity :AppCompatActivity(){

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_module_aactivity)

//        ARouter.getInstance().build("/a/module_a_activity").navigation()
    }

}